book/src/dev/release-criteria.md

```diff
@@ -0,0 +1,57 @@
+# Go/No-Go Release Criteria
+
+## Alpha Release
+
+In the first alpha release, we want Zebra to participate in the network and replicate the Zcash chain state. We also want to validate proof of work and the transaction merkle tree, and serve blocks to peers.
+
+### System Requirements
+
+_TBD_
+
+### Build Requirements
+
+_TBD_
+
+### Supported Platforms
+
+_TBD_
+
+### Go/No-Go Criteria
+
+- Zebrad Functionality
+    - [ ] zebrad can sync to mainnet tip
+        - [ ] under perfect network conditions (within 2 - 3 hours)
+        - [ ] under reasonable network conditions (in under 5 hours)
+        - [ ] under sub-optimal network conditions (?)
+    - [ ] zebrad can keep up with the mainnet tip after the initial sync
+        - [ ] under perfect network conditions
+        - [ ] under reasonable network conditions
+        - [ ] under sub-optimal network conditions
+    - [ ] zebrad can validate proof of work
+    - [ ] zebrad can validate the transaction merkle tree
+    - [ ] zebrad can serve blocks to peers
+- Zebrad Performance
+    - [ ] _TBD_
+- Testing
+    - [ ] CI Passes
+        - [ ] Unit tests pass reliably
+        - [ ] Property tests pass reliably
+        - [ ] Acceptance tests pass reliably
+- Network Readiness
+    - [ ] zebrad can interact with other nodes
+    - [ ] We have a mechanism to deal with misbehaving zebrad nodes
+- Implementation and Launch
+    - [ ] All blocker bugs have been fixed
+    - [ ] Release notes are available
+    - [ ] Known issues are clearly documented
+    - [ ] Users can access the documentation to deploy zebrad nodes
```

Comment:
Are release notes, known issues documentation, and deployment documentation essential for the first alpha?

Reply:
I think known issues should be included at least so users know what to expect. Happy to discuss further

Reply:
Sounds good - but since many users will just build the `main` branch, the list of known issues will change over time.

So let's include a link to a GitHub query - something like "severe bug" or "release bug".

We don't rate the severity of our bugs yet. But we can't just send people to the full list of bugs, because it contains a lot of bugs they probably won't care about.

Reply:
I've added a link to the existing issues labelled with "C-bug" that are on the alpha release milestone. We should definitely add some additional priority and severity labels to our repo after alpha to help with triaging external bug reports and help us define what constitutes a blocking bug for future releases.